2024-07-18 11:41:54,589 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-18 11:42:54,651 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-18 11:43:54,756 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-18 11:44:54,821 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-18 11:45:54,912 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-19 11:00:12,803 ERROR scheduler Skipped queueing hrms.hr.doctype.interview.interview.send_interview_reminder because it was found in queue for erp.grupoatu.com
2024-07-19 11:00:12,908 ERROR scheduler Skipped queueing frappe.automation.doctype.reminder.reminder.send_reminders because it was found in queue for erp.grupoatu.com
2024-07-19 11:00:12,911 ERROR scheduler Skipped queueing frappe.monitor.flush because it was found in queue for erp.grupoatu.com
2024-07-19 11:00:12,918 ERROR scheduler Skipped queueing frappe.utils.global_search.sync_global_search because it was found in queue for erp.grupoatu.com
2024-07-19 11:00:12,920 ERROR scheduler Skipped queueing frappe.email.doctype.email_account.email_account.notify_unreplied because it was found in queue for erp.grupoatu.com
2024-07-19 11:00:12,928 ERROR scheduler Skipped queueing frappe.email.queue.flush because it was found in queue for erp.grupoatu.com
2024-07-19 12:01:19,688 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-19 12:02:19,793 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-19 12:03:19,853 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-19 12:04:19,910 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 644, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 73, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 110, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 123, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 358, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 711, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")
2024-07-25 17:35:18,143 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:36:18,206 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:40:18,635 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:41:18,700 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:42:18,751 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:43:18,807 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:44:18,903 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:45:18,956 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:46:19,020 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:47:19,071 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:48:19,191 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:49:19,213 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:50:19,234 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:51:19,329 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:52:19,376 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:53:19,497 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:54:19,521 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:55:19,589 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:56:19,682 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:57:19,792 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:58:19,915 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 17:59:20,001 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:00:20,125 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:01:20,171 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:02:20,293 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:03:20,379 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:04:20,443 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:05:20,562 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:06:20,646 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:07:20,726 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:08:20,812 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:09:20,864 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:10:20,984 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:11:21,081 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:12:21,189 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:13:21,305 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:14:21,433 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:15:21,508 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:16:21,617 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:17:21,735 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:18:21,854 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:19:21,973 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:20:22,052 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:21:22,145 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:22:22,207 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:23:22,313 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:24:22,332 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:25:22,433 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:26:22,554 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:27:22,673 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:28:22,779 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:29:22,825 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:30:22,942 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:31:23,059 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:32:23,138 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:33:23,258 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:34:23,367 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:35:23,486 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:36:23,574 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:37:23,667 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:38:23,753 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:39:23,882 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:40:23,937 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:41:24,055 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:42:24,122 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:43:24,187 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:44:24,248 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:45:24,365 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:46:24,482 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:47:24,543 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:48:24,659 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
2024-07-25 18:49:24,764 ERROR scheduler Exception in Enqueue Events for Site erp.grupoatu.com
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events
    if job_type.enqueue():
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 71, in enqueue
    if self.is_event_due() or force:
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 90, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "/home/frappe/frappe-bench/apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 116, in get_next_execution
    "All": f"*/{(frappe.get_conf().scheduler_interval or 240) // 60} * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'
